Tests/TemplatesTests/StringsTests.swift

```diff
@@ -33,6 +33,19 @@ class StringsTests: XCTestCase {
     ]
   }
 
+  let variationsObjC: VariationGenerator = { name, context in
+    guard name == "localizable" else { return [(context: context, suffix: "")] }
+
+    return [
+      (context: context,
+       suffix: ""),
+      (context: try StencilContext.enrich(context: context, parameters: ["noComments"]),
+       suffix: "-no-comments"),
+      (context: try StencilContext.enrich(context: context, parameters: ["headerName=L10N"]),
```

Comment:
Pretty sure this should become `"headerName=localizable-headerName.h"`? (if we're ever going to test this)

Reply:
'L10N' is different from the default 'Localizable', so I think this exercises the feature, no?

Is there already a scaffolding for testing the content of a generated file?  If so, I would test that the generated .M file includes the right value in its `#include ...` line.

Reply:
Well no that's for a later PR (hopefully). But just for us, for clarity of the tests (and maybe for our users), it probably should have a correct value.

Note: it should at least be `L10n.h`, as the template doesn't add the `.h` anymore.

Reply:
good catch (on lack of ".h")!  

and I'll switch to `headerName=headerName-from-parameter.h` to make it clear what's going on.